src/Middleware/Rewrite/src/ApacheModRewrite/Condition.cs

```diff
@@ -1,18 +1,25 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
📝 The encoding changes are concerning

Reply:
My changes are removing the BOMs which I assume is correct 🤷

Reply:
C# source files should have a byte order mark. It looks like the repository has an incorrectly-configured **.editorconfig** file which is causing these changes to appear.

Reply:
➡️ Filed #28697 to fix this